Views/Rooms/Index.cshtml

```diff
@@ -0,0 +1,127 @@
+@model IEnumerable<Room>
+
+<h1>@Localizer["All Rooms"]</h1>
+
+<div>
+    <div class="input-group mb-3">
+        <input id="input-name-search" type="text" class="form-control" placeholder="@Localizer["Search by name..."]" aria-label="@Localizer["Search by name..."]">
+        <button class="btn btn-outline-secondary" type="button" onclick="searchName(this)">@SharedLocalizer["Search"]</button>
+    </div>
+    
+    <div class="input-group mb-3">
+        <label class="input-group-text" for="faculty-filter-switch">@Localizer["Filter by faculty"]</label>
+        <select id="faculty-filter-switch" asp-items="Html.GetEnumSelectList<Faculty>()" onchange="filterFaculty(this)" class="form-select" style="max-width: 128px">
+            <option value="-1">@SharedLocalizer["All"]</option>
+        </select>
+    </div>
+</div>
+
+<table class="table table-hover" style="cursor: pointer;">
+    <thead>
+    <tr class="user-select-none">
+        <th hidden scope="col">Id</th>
+        <th scope="col" id="table-head-name" onclick="sortBy('name')">
+            @SharedLocalizer["Name"]
+            <i class="bi bi-dash-lg pe-1"></i>
+        </th>
+        <th scope="col" id="table-head-faculty" onclick="sortBy('faculty')">
+            @SharedLocalizer["Faculty"]
+            <i class="bi bi-dash-lg pe-1"></i>
+        </th>
+        <th scope="col" id="table-head-maxDuration" onclick="sortBy('maxDuration')">
+            @SharedLocalizer["MaxDuration"]
+            <i class="bi bi-dash-lg pe-1"></i>
+        </th>
+        <th scope="col" id="table-head-maxParticipants" onclick="sortBy('maxParticipants')">
+            @SharedLocalizer["MaxParticipants"]
+            <i class="bi bi-dash-lg pe-1"></i>
+        </th>
+    </tr>
+    </thead>
+    <tbody>
+    @foreach (var room in Model)
+    {
+        <tr>
+            <th hidden scope="row">@room.Id</th>
+            <td>@room.Name</td>
+            <td>@room.Faculty</td>
+            <td>@room.MaxDuration</td>
+            <td>@room.MaxParticipants</td>
+            @* <td> *@
+            @*     <a type="submit" asp-action="Remove" asp-route-id="@room.Id" class="btn btn-primary"><i class="bi bi-trash"></i></a> *@
+            @* </td> *@
+        </tr>
+    }
+    </tbody>
+</table>
+
+@section Scripts {
```

Comment:
ich finde es hier nicht gut, dass wir JavaSvript logic in HTML haben, vlt sollten wir es hier auslagern :)

Reply:
Können wir irgendwann mal machen. Das @section ist aber genau dafür gedacht, dass das clean bleibt und immer an der richtigen Stelle reingeladen wird und da die Logik genau und nur genau zu dieser Seite gehört macht es atm. Sinn das hier zu lassen, da da nix super großes passiert.

Reply:
oki :)